src/Illuminate/Support/Str.php

```diff
@@ -234,6 +234,17 @@ public static function lower($value)
     }
 
     /**
+     * Prevent widow in a string.
+     *
+     * @param  string  $value
+     * @return string
+     */
+    public static function no_widow($value)
+    {
+        return preg_replace('|([^\s])\s+([^\s]+)\s*$|', '$1&nbsp;$2', $value);
```

Comment:
I would change &nbsp; with a blank space, this implementation only makes sense for HTML outputs.

Reply:
@jmarcher Thank you for checking it out! The `&nbsp;` is used to prevent a actual blank space between the last two words, which makes sure the last word isn't sitting alone on a bottom line.